src/utils/sort.js

```diff
@@ -0,0 +1,13 @@
+export const sort  = (products,filters) => {
+  const {sort} = filters;
+  switch (sort) {
+    case "increasing":
```

Comment:
Suggestion : You could have named these cases to be like this LOW_TO_HIGH and HIGH_TO_LOW .

Reply:
Thanks, I should change it to allUpperCase but names `increasing/decreasing` are pretty self explanatory i  guess.